src/core/core.scale.js

```diff
@@ -716,7 +716,7 @@ module.exports = Element.extend({
 	},
 
 	// Actually draw the scale on the canvas
-	// @param {rectangle} chartArea : the area of the chart to draw full grid lines on
+	// @param {ILayoutItem} chartArea : the area of the chart to draw full grid lines on
```

Comment:
I don't think it's a `ILayoutItem` but a regular TRBL `object`. Also, `:` should be `-`

Reply:
done

Reply:
I've gone ahead and changed this since `object` is definitely correct and there's debate over `ILayoutItem`. The reason I think it's an `ILayoutItem` is that it gets set in `core.layouts.js` `update` method and has the properties that an `ILayoutItem` defines: https://github.com/chartjs/Chart.js/pull/6033/files#r254161515

Reply:
> ... and has the properties that an `ILayoutItem` defines

It only has `top`, `left`, `right` and `bottom` in common with `ILayoutItem`, so it's definitely not a [`ILayoutItem`](/chartjs/Chart.js/blob/6e9e52afacae801f6247a321c4fe2e01afc0eaab/src/core/core.layouts.js#L69) (the title, legend, scale, etc. are layout items).